styles/global.css

```diff
@@ -2,3 +2,15 @@
   font-family: Cascinarampina;
   src: url("/cascinarampina.ttf");
 }
+
```

Comment:
we shouldn't need this css, the burger menu you can make a component for that, also I think theme-ui exports a burger menu button look, maybe we can use that one. (I don't mind using whatever you wish)
https://theme-ui.com/components/menu-button